Sources/PackageManagerDocs/Documentation.docc/Templates.md

```diff
@@ -0,0 +1,458 @@
+#  Creating Package Templates
+
+This guide provides a brief overview of Swift Package Manager templates, describes how a package can make use of templates, and shows how to get started writing your own templates.
```

Comment:
```suggestion
Create a template plugin for Swift Package Manager to generate custom Swift packages based on the custom inputs that you define.
```

^ The first sentence in the article is called an "Abstract" and we generally want it to be a short, imperative sentence that tells the reader what they'll be learning if they jump in and read it. The abstract is presented alongside the title to the create a sort of description of what they'll learn and what core problem its solving.

I suspect there could easily be better title and abstract choices here, but wanted to at least suggest a placeholder that was closer to what we should be aiming for.